client/extensions/wp-job-manager/components/setup/page-setup/index.jsx

```diff
@@ -0,0 +1,168 @@
+						'existing page on your site, you can skip this step and take a look at {{shortcodeRef}}shortcode ' +
+						'support documentation{{/shortcodeRef}} for detailed instructions.){{/p}}',
+						{ components: {
+							em: <em />,
+							p: <p />,
+							shortcodes: (
+								<ExternalLink
+									icon={ true }
+									target="_blank"
+									href="https://codex.wordpress.org/Shortcode"
+								/>
+							),
+							shortcodeRef: (
+								<ExternalLink
+									icon={ true }
+									target="_blank"
+									href="https://wpjobmanager.com/document/shortcode-reference/"
+								/>
+							)
+						} }
+					) }
+
+					<div className="page-setup__pages">
+						<form>
+							<FormFieldset>
+								<ReduxFormToggle
+									name="createPostJob" />
+								<ReduxFormTextInput
+									disabled={ ! createPostJob }
+									name="postJobTitle" />
+
+								<FormSettingExplanation>
+									{ translate(
+										'Creates a page that allows employers to post new jobs directly from a page on your website, ' +
+										'instead of requiring them to log in to an admin area. If you\'d rather not allow this -- ' +
+										'for example, if you want employers to use the admin dashboard only -- you can uncheck ' +
+										'this setting.'
+									) }
+								</FormSettingExplanation>
+							</FormFieldset>
+
+							<FormFieldset>
+								<ReduxFormToggle
+									name="createDashboard" />
+								<ReduxFormTextInput
+									disabled={ ! createDashboard }
+									name="dashboardTitle" />
+
+								<FormSettingExplanation>
+									{ translate(
+										'Creates a page that allows employers to manage their job listings directly from a page on your ' +
+										'website, instead of requiring them to log in to an admin area. If you want to manage all ' +
+										'job listings from the admin dashboard only, you can uncheck this setting.'
+									) }
+								</FormSettingExplanation>
+							</FormFieldset>
+
+							<FormFieldset>
+								<ReduxFormToggle
+									name="createJobs" />
+								<ReduxFormTextInput
+									disabled={ ! createJobs }
+									name="jobsTitle" />
+
+								<FormSettingExplanation>
+									{ translate(
+										'Creates a page where visitors can browse, search, and filter job listings.'
+									) }
+								</FormSettingExplanation>
+							</FormFieldset>
+						</form>
+					</div>
+				</CompactCard>
+
+				<CompactCard>
+					<a
+						className="page-setup__skip"
+						href="#">
+						{ translate( 'Skip this step' ) }
+					</a>
+					<Button primary
+						className="page-setup__create-pages"
+						disabled={ ( ! createPostJob && ! createDashboard && ! createJobs ) }>
+						{ translate( 'Create selected pages' ) }
+					</Button>
+				</CompactCard>
+			</div>
+		);
+	}
+}
+
+const selector = formValueSelector( form );
+
+const mapStateToProps = state => {
```

Comment:
This could be written as `state => selector( state, 'createDashboard', 'createJobs', 'createPostJob' );`.  
Just putting that out there - it's up to you to decide what's more readable for you.

Reply:
Shorter, terser code wins for me almost every time. 😄